rusk-profile/src/lib.rs

```diff
@@ -57,7 +57,10 @@ fn file_stem(p: &Path) -> Option<&str> {
 }
 
 pub fn get_rusk_profile_dir() -> Result<PathBuf, io::Error> {
-    if let Some(mut dir) = home_dir() {
+    if let Some(env_var) = option_env!("RUSK_PROFILE_PATH") {
+        fs::create_dir_all(env_var.clone())?;
```

Comment:
If the env var is set, it is safe to assume the dir is already created by the user. Besides, there may be a conflict with existing dirs.

Suggestion: remove this line